brave/brave-core
# Modify Rewards Balance P3A question: add "zero balance" bucket

Resolves brave/brave-browser#21091

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

Rewards P3A browser tests should pass.

0. Perform fresh install of browser with fresh config
1. Enable Brave Rewards, enable Ads and/or Auto-Contribution
2. Verify that the wallet balance is 0.
3. Visit `brave://local-state`. Verify that `Brave.Rewards.WalletBalance.3` is set to 2 under the P3A object.
